src/CrowTypes.ts

```diff
@@ -54,6 +54,7 @@ export interface RadarInterface {
   endpoint: string; // URL template, some variables are interpolated. Example: 'https://opendata.meteo.be/ftp/observations/radar/vbird/{odimCode}/{yyyy}/{odimCode}_vpts_{yyyymmdd}.txt'
   vptsFileFormat: VPTSFileFormat;
   heights: number[]; // Data is available at the following heights
+  showOnMap?: boolean; // If false, the radar will not be displayed on the map
```

Comment:
```suggestion
  showOnMap?: boolean; // If false, the radar will not be displayed on the map (default true)
```

Note is the `?` intentional?